src/test/java/uk/gov/companieshouse/overseasentitiesapi/validation/OwnersAndOfficersDataBlockValidatorTest.java

```diff
@@ -113,16 +113,19 @@ void testNoErrorReportedForSomeBeneficialOwnersIdentifiedWithOnlyBeneficialOwner
         assertFalse(errors.hasErrors());
     }
 
+    // TODO roe-1640 AC are that both at least one BO and MO must be present if either one is missing an error is logged and returned
```

Comment:
see previous comment about the TODO comment

Reply:
As above